backend/app/modules/user/antisybil/service/holonym.py

```diff
@@ -0,0 +1,44 @@
+from flask import current_app as app
+
+from eth_utils.address import to_checksum_address
+
+import json
+from typing import List, Optional, Tuple
+
+from app.extensions import db
+from app.infrastructure import database
+from app.context.manager import Context
+from app.pydantic import Model
+from app.exceptions import UserNotFound
+
+
+class HolonymAntisybil(Model):
+    def get_sbt_status(
+        self, _: Context, user_address: str
+    ) -> Optional[Tuple[bool, List[str]]]:
```

Comment:
I'd suggest adding a wrapper for the results, e.g

```
@dataclass
class HolonymAntisybilDTO:
   has_sbt: bool
   sbt_details: List[str]
```
and then return its object from the service;

we'll get rid of these tuples that aren't quite readable as return types when we retrieve or annotate'em

Reply:
I've reworked both passport and holonym return types to use DTOs